packages/react-next/.storybook/preview.js

```diff
@@ -4,13 +4,14 @@ import { configure, addParameters, addDecorator } from '@storybook/react';
 import { withA11y } from '@storybook/addon-a11y';
 import { withPerformance } from 'storybook-addon-performance';
 import { withKnobs } from '@storybook/addon-knobs';
-import { withThemeProvider } from '@fluentui/storybook';
+import { withThemeProvider, withStrictMode } from '@fluentui/storybook';
 import { withKeytipLayer } from './decorators';
 
 addDecorator(withA11y());
 addDecorator(withPerformance);
 addDecorator(withKnobs({ escapeHTML: false }));
 addDecorator(withThemeProvider);
+addDecorator(withStrictMode);
```

Comment:
actually, we should default to be enabled for `react-next` package

Reply:
I think it should be opt-in to avoid confusing people. One of the things strict mode does is [run certain things twice](https://reactjs.org/docs/strict-mode.html#detecting-unexpected-side-effects), and if someone isn't expecting that, it could confuse them while trying to debug something.

Reply:
I don't feel too strongly about this. But i still think it's overall better to turn this on by default. That way dev can spot issues with strict mode on as early as possible during development phase with less chance of missing. After all, if we have code that doesn't work well with strict mode, we need to get those fixed.

Reply:
For obvious new issues, I think a better way to prevent them from being introduced is with lint rules. [`eslint-plugin-react`](https://github.com/yannickcr/eslint-plugin-react) already has rules for preventing UNSAFE_ and findDOMNode, so we just need to enable them. (We already have a rule about no string refs enabled.) 

For the non-obvious issues with side effects, which the double rendering is meant to catch, I'm still concerned that it would cause confusion when debugging more often than it would be helpful.

Reply:
ok, i'm fine starting with this. can always update when we find fit